onnxutils/onnxtk/common/utils.py

```diff
@@ -145,31 +145,31 @@ def get_producer():
     """
     Internal helper function to return the producer
     """
-    from ... import __producer__
+    from onnxmltools import __producer__
```

Comment:
Unsure if this is the right package to draw from, but when calling from the _init_.py file in onnxtk, the "domain" causes test failures due to the disparity between the location of the tests and the location of the helper functions. @wenbingl, thoughts?

Reply:
Resolved to using native package __producer__ with ".." in recent update.